### DISCUSSION TITLE
Feature Request: Bandcamp Feed Block Ads ( Only New Albums)

### DISCUSSION THREAD

**Discussion by freeload101** [cvzi/Bandcamp-script-deluxe-edition#245] [Ideas]
https://www.reddit.com/r/uBlockOrigin/comments/x7ufvw/bandcamp_feed_block_ads_only_new_albums/

Bandcamp Feed Block Ads ( Only New Albums)

```
! 2022-09-07 https://bandcamp.com
bandcamp.com##li.fp.story > .track_play_hilite.collection-item-container.story-innards
bandcamp.com##li.df.story > .track_play_hilite.collection-item-container.story-innards
bandcamp.com##li.np.story > .story-innards > .story-body
bandcamp.com##li.np.story > .story-innards
```

**Comment by cvzi** [comment]
Yeah sure, I can add that

**Comment by cvzi** [comment]
I added it as an option. It is disabled by default, there is a new option in the settings of the script to enable it.

I didn't use your filter. Instead it removes all `.story` that don't have the class `.nr` ([relevant commit](https://github.com/cvzi/Bandcamp-script-deluxe-edition/commit/bdeaaf49ee4638e2f99ece737ac371ac546cb9ea#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R5804)). I assume "nr" stands for "new release", so I hope that works as good as your filter.
